server/src/core/system-lock/tests/system-lock.test.ts

```diff
@@ -16,91 +16,109 @@ describe('SystemLock tests', () => {
     await dbUtils.tearDown();
   });
 
-  beforeEach(async () => {
-    await dbUtils.resetCollectionWith(systemLocks);
-  });
+  // beforeEach(async () => {
+  //   await dbUtils.resetCollectionWith(systemLocks);
+  // }, 10000);
 
   describe('lock', () => {
+    beforeAll(async () => {
+      await dbUtils.resetCollectionWith(systemLocks);
+    }, 10000);
+
     test('should set locked:true on lock record', async () => {
       const lock = new SystemLockManager('lock2', dbUtils.getCollection());
       await lock.lock();
       const record = await dbUtils.getCollection().findOne({ _id: 'lock2' });
       expect(record.locked).toBe(true);
-    });
+    }, 10000);
 
     test('should create lock record with locked:true if record does not exist', async () => {
-      await dbUtils.dropCollection();
       const lock = new SystemLockManager('lock2', dbUtils.getCollection());
       await lock.lock();
       const record = await dbUtils.getCollection().findOne({ _id: 'lock2' });
       expect(record.locked).toBe(true);
-    });
+    }, 10000);
```

Comment:
This test does not reflect its description. It's supposed to test locking a lock whose db record does not exist. That's why it started by dropping the collection, to ensure that the lock did not exist. So now this test is not testing the desired behaviour. Either replace `lock2` with a non-existent lock id, or restored the call to `dbUtils.dropCollection()`. I'd prefer not changing the semantics of the a test for a behaviour that's not supposed to be affected by your feature.